corehq/apps/users/migrations/0032_add_download_reports_permission.py

```diff
@@ -0,0 +1,26 @@
+import traceback
+from django.db import migrations
+
+from corehq.apps.couch_sql_migration.tests.test_migration import call_command
```

Comment:
incorrect import:
```suggestion
from django.core.management import call_command
```

Reply:
I committed this since it was breaking the staging deploy

Reply:
sorry about that -- thanks!